test/integration/browser_type/connect_cdp_test.exs

```diff
@@ -1,7 +1,121 @@
 defmodule Playwright.BrowserType.ConnectCDPTest do
-  use Playwright.TestCase, async: true
+  @remote_debug_port "9222"
 
-  # test_connect_to_an_existing_cdp_session
-  # test_connect_to_an_existing_cdp_session_twice
-  # test_conect_over_a_ws_endpoint
+  use Playwright.TestCase, async: true, args: ["--remote-debugging-port=#{@remote_debug_port}"]
+
+  alias Playwright.Browser
+  alias Playwright.BrowserContext
+  alias Playwright.BrowserType
+  alias Playwright.Page
+
+  @tag :skip
+  test "can only connect to CDP session if using chromium client" do
+    # For some reason this still launches a chromium browser... I could not
+    # figure out why.
+
+    browser = Playwright.launch(:firefox)
```

Comment:
@coreyti any thoughts about how to better test this error case? I couldn't quite figure out why but it appears that no matter whether I call `Playwright.launch()` with (ie `:webkit` or `:firefox`) I still get a chromium `%Playwright.Browser{}` back.

Perhaps this is just something to note and move on from since technically this library only supports chromium at the moment.